internal/feature/clientcert.go

```diff
@@ -11,4 +11,7 @@ var clientCert = Feature{
 	ProviderAWS: Capability{
 		MinVersion: "16.0.1",
 	},
+	ProviderOpenStack: Capability{
+		MinVersion: "20.0.0-alpha1",
```

Comment:
We don't have any release version on OpenStack so I'm not sure if this is necessary.

Reply:
We don't have any kyverno policies on OpenStack at the moment.

Reply:
I got rid of the entire thing as it was not needed anymore. The only clusters in an older version we still run are 2 in `viking` and we can create clientCerts for those using the `--insecure-namespace` flag.